### PR TITLE
fix: add trace_id span_id fields to java jaeger example

### PR DESCRIPTION
**Pull request by korniltsev**

### CODE REVIEW

**Comment by Rperry2174** [comment]
Looks great.. Can we tweak the log message to also have something unique that will allow us to identify it as java? Maybe add rideshare-java to the message somewhere:

The query bar is populated via this regexp:
image

so in order for us to link directly to the _java_ app we need to have a slightly different regexp that is specific to java:

image

File: examples/tracing/jaeger/java/rideshare/src/main/resources/application.properties [hunk 1]
```diff
@@ -1 +1,3 @@
 server.port=5000
+logging.pattern.level=%clr(%5p) %clr([ trace_id=%X{traceId:-} span_id=%X{spanId:-} ])
```
Comment (Rperry2174) [inline examples/tracing/jaeger/java/rideshare/src/main/resources/application.properties:2]:
```suggestion
logging.pattern.level=%clr(%5p) %clr([ trace_id=%X{traceId:-} java_span_id=%X{spanId:-} ])
```

**Comment by korniltsev** [comment]
looks good

**Review by Rperry2174 — APPROVED** [review]
LGTM -- it would be nice if we only logged this for the _root_ span instead of all spans, but not sure if thats possible here. Fwiw here is how we did it in Go:
https://github.com/pyroscope-io/pyroscope/pull/1124/files

**Pull request merged**